main.py

```diff
@@ -37,26 +37,37 @@
 # acquire_line(speed = 20)
 # rot_motion()
 # line_follower(move_degrees = 1000, speed = 20)
-# gyro_turn(input_angle = 90, relative = False)
+# gyro_turn(input_angle = 90, relative = False, timeout = 6, left_or_right = OneWheelTurn.BOTH)
 # grind(left_speed = 20, right_speed = 20, run_seconds=3)
 #vrooom()
 ###############################################################
 
+class OneWheelTurn:
+    BOTH = 0
+    LEFT = 1
+    RIGHT = 2
+
+def motor_front_move():
+    motor_front_left.set_degrees_counted(0)
+    motor_front_right.set_degrees_counted(0)
+    motor_front_left.run_for_degrees(-80, speed=100)
+    motor_front_right.run_for_degrees(4500, speed=100)
+    motor_front_right.run_for_degrees(-4500, speed=100)
+
 def tuning():
     grind()
 
 def test_trip():
-    #two_wheel_move(left_degrees=-400, right_degrees=-100, speed = 40)
-    #acquire_line(speed=20)
-    #line_follower(move_degrees=300, speed=20)
-    #line_follower(move_degrees=800, speed=40)
-    #line_follower(move_degrees=790, speed=30)
-    #gyro_turn(input_angle = 120, relative = False)
-    #grind(left_speed=-30, right_speed = -30, run_seconds=20
-    #make_mark()
+    gyro_turn(input_angle = 90, relative = False, timeout = 6, left_or_right = OneWheelTurn.RIGHT)
 
 def the_trip_with_the_crates():
-    gyro_turn(20, relative = False)
+    two_wheel_move(left_degrees = 362, right_degrees = 272, speed=25)
+    acquire_line(speed = 20)
+    line_follower(move_degrees = 600, speed = 20, gain = 0.6)
```

Comment:
@FortressBen @EricGamer1127 see above comment: no spaces around the equals when you call methods and pass a parameter